docs/templates/getting-started/functional-api-guide.md

```diff
@@ -24,9 +24,9 @@ from keras.models import Model
 inputs = Input(shape=(784,))
 
 # a layer instance is callable on a tensor, and returns a tensor
-x = Dense(64, activation='relu')(inputs)
-x = Dense(64, activation='relu')(x)
-predictions = Dense(10, activation='softmax')(x)
+hidden_layer_1 = Dense(64, activation='relu')(inputs)
```

Comment:
These variables are symbolic tensors, not layer instances, so I think it would be misleading to name them "*_layer".

Reply:
Fair. Better ideas? "activations" might be better though it doesn't express the symbolic part. I'm not super familiar with how things are named in Keras.

Reply:
"activation" is used to denote activation functions specifically. These objects are "tensors", or perhaps "outputs".